src/modules/clientFiles/components/FileModal.tsx

```diff
@@ -2,7 +2,9 @@ import ImageNotSupportedIcon from '@mui/icons-material/ImageNotSupported';
 import {
```

Comment:
This should probably now be moved out of the ClientFiles module, but if it's helpful, I'm happy to postpone that change until you've reviewed the actual changes within this file, since it's probably easier to read the diff before moving it.